Tasks/DuffleInstallerV0/src/duffleinstaller.ts

```diff
@@ -39,7 +39,7 @@ async function downloadDuffle(version: string): Promise<string> {
     }
 
     const dufflePath = path.join(cachedToolPath, DuffleToolName + getExecutableExtension());
-    fs.chmod(dufflePath, '777');
+    fs.chmodSync(dufflePath, '777');
```

Comment:
Is this required as part of Node 10 upgrade?

Reply:
Yup. In Node6 chmod was synchronous by default